K2Bridge/Factories/BucketFactory.cs

```diff
@@ -46,7 +52,78 @@ public static DateHistogramBucket CreateDateHistogramBucketFromDataRow(DataRow r
                     continue;
                 }
 
-                dhb.Aggs[clmn.ColumnName] = new System.Collections.Generic.List<double>() { Convert.ToDouble(row[clmn.ColumnName]) };
+                // Create new Regex to match the ID%PERCENTILE - example: 1%50.0
+                // var regex = new Regex(@"^(\d+)%{1}(100\.0|[0-9]?[0-9]\.[0-9]{1})$");
+                // var match = regex.Match(clmn.ColumnName);
+                var columnNameInfo = clmn.ColumnName.Split('%');
+
+                if (columnNameInfo.Length > 1)
+                {
+                    // key%metric%value1%value2%keyed
+                    var key = columnNameInfo[0];
+                    var metric = columnNameInfo[1];
+                    var queryValues = columnNameInfo[2..^1];
```

Comment:
I'd be happy to get a comment here on line 65, 66 what exactly is done

Reply:
done